tests/unit/aroi/test_aroi_validation.py

```
@@ -551,5 +551,735 @@ def test_fingerprint_sets_populated(self):
+        self.assertEqual(s['validation_summary']['v3_tier'], 'none')
+        self.assertFalse(s['validation_summary']['is_v3_adopter'])
+
+    def test_schema_version_handshake_unknown(self):
+        """A.1: schema version outside tested set logs a warning, doesn't crash."""
+        from allium.lib.aroi_validation import (
+            check_schema_version, _warned_schema_versions
+        )
+        _warned_schema_versions.clear()  # Test isolation
+        warnings = []
+
+        def capture(msg):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `capture`

Add return type annotation: `None`

(ANN202)